packages/account-v2/src/pages/ManualUploadContainer/ManualUploadContainer.tsx

```diff
@@ -24,34 +26,56 @@ export const ManualUploadContainer = ({
         selectedDocument as TManualDocumentTypes
     );
 
-    const documentUploadSchema = getManualFormValidationSchema(
-        selectedDocument as TManualDocumentTypes,
-        isExpiryDateRequired
-    );
-
-    const selfieUploadSchema = getSelfieValidationSchema();
-
-    const manualUploadSchema = object({
-        ...documentUploadSchema.fields,
-        ...selfieUploadSchema.fields,
-    });
-
-    type TManualUploadFormData = InferType<typeof manualUploadSchema>;
-
-    const [formData, setFormData] = useState<Partial<TManualUploadFormData>>({});
+    const [formData, setFormData] = useState<Partial<TManualDocumentUploadFormData>>({});
     const [shouldUploadSelfie, setShouldUploadSelfie] = useState(false);
+    const { upload } = useDocumentUpload();
 
     if (isLoading || poiService === null) {
         return <Loader />;
     }
 
+    const processDocuments = (
+        values: TManualDocumentUploadFormData,
+        item: ReturnType<typeof getUploadConfig>[number]
+    ): Parameters<typeof upload>[0] => {
+        const { documentType, pageType } = item;
+        return {
+            document_id: values.documentNumber,
+            document_issuing_country: countryCode,
+            document_type: documentType,
+            expiration_date: values?.documentExpiry?.toString() ?? undefined,
+            file:
+                documentType === MANUAL_DOCUMENT_TYPES.selfieWithID
+                    ? (values.selfieWithID as File)
+                    : (values[pageType] as File),
+            lifetime_valid: isExpiryDateRequired ? 0 : 1,
+            page_type: documentType === MANUAL_DOCUMENT_TYPES.selfieWithID ? 'photo' : pageType,
+        };
+    };
+
+    const handleSubmit = async (values: TManualDocumentUploadFormData) => {
+        const uploadDocumentConfig = getUploadConfig(selectedDocument as TManualDocumentTypes);
+
+        try {
+            await uploadDocumentConfig.reduce(async (promise, item) => {
+                await promise;
+                const payload = await processDocuments(values, item);
+                await upload(payload);
+            }, Promise.resolve());
+            onDocumentSubmit();
+        } catch (error) {
+            // eslint-disable-next-line no-console
+            console.error(error);
```

Comment:
Is it okay to console the error, instead of showing a popup or message ?

Reply:
We dont have an Error scenario yet. This has been raised and a figma design is design created by UX team